src/ListingPage.php

```diff
@@ -47,6 +47,9 @@ private function update_context(): void
         $this->context['page_category'] = is_home() ? 'News' : 'Listing Page';
         $this->context['og_type'] = isset($this->context['author']) ? 'profile' : 'website';
 
+        // Set layout (grid or list)
+        $this->set_layout();
```

Comment:
Does it make sense to create a single-line function that is not reused, either?
Maybe we can just go with:
```suggestion
        $this->context['layout'] = $_GET['layout'] ?? 'list';
```

Reply:
Updated!